src/App.js

```diff
@@ -0,0 +1,22 @@
+import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Bookstore from './components/Bookstore';
+import Navbar from './components/Navbar';
+
+function App() {
+  return (
+    <div className="App">
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Bookstore />}>Home</Route>
+          <Route path="/category" element={<Bookstore />}>Home</Route>
```

Comment:
### **Optional**

- [ ] Kindly create and add a Categories component to your project to make it more explicit.